neighbors.cc

```diff
@@ -16,6 +16,7 @@
 #include <iostream>
 #include "gflags/gflags.h"
 #include "init.h"
+#include <cstring>
```

Comment:
Please keep library `#include`s grouped at the top of the list.

Reply:
Thanks, I'll make that change.